charts/revere/values.yaml

```diff
@@ -0,0 +1,211 @@
+# -- (object) Required; contents of revere.yaml to be given to the application
+config:
+
+# Global values used by this chart
+global:
+  # -- (string) What version of the application to deploy
+  applicationVersion: latest
+
+# name -- A name for the deployment that will be substituted into resource definitions
+name: revere
+
+imageConfig:
+  # -- (string) Image repository
+  repository: us-central1-docker.pkg.dev/dsp-artifact-registry/revere/revere
+  # -- (string) Image tag
+  # @default -- global.applicationVersion
+  tag: null
+  # -- (string) When to pull images
+  imagePullPolicy: Always
+
+# -- (number) Number of replicas for the deployment
+replicas: 3
+
+# -- (map) Annotations for application pods
+annotations: {}
+
+# -- (string) Optional override for the pod's runtimeClass
+runtimeClassName:
+
+resources:
+  requests:
+    # -- (string) Number of CPU units to request for the deployment
+    cpu: 500m
+    # -- (string) Memory to request for the deployment
+    memory: 4Gi
+  limits:
+    # -- (string) Number of CPU units to limit the deployment to
+    cpu: 500m
+    # -- (string) Memory to limit the deployment to
+    memory: 4Gi
+
+probes:
+  readiness:
+    # -- (boolean) If the readiness probe should be enabled
+    enabled: false
+    # -- (object) Spec for the readiness probe
+    spec:
+  liveness:
+    # -- (boolean) If the liveness probe should be enabled
+    enabled: false
+    # -- (object) Spec for the liveness probe
+    spec:
+  startup:
+    # -- (boolean) If the startup probe should be enabled
+    enabled: false
+    # -- (object) Spec for the startup probe
+    spec:
+
+secrets:
+  gcpServiceAccount:
+    workloadIdentity:
+      # -- (boolean) If workloadIdentity should be used for the GCP SA
+      enabled: false
+      # -- (string) ID of the GCP SA to use
+      accountName:
+      # -- (string) ID (not the number) of the GCP project the SA is in
+      projectId:
+    secretsManager:
+      # -- (boolean) If secretsManager should be used for the GCP SA
+      enabled: false
+      # -- (string) Path within Vault where the SA's key is stored
+      vaultPath:
+      # -- (string) Field name within the secret for the SA's key
+      vaultKey:
+      # -- (string) Encoding of the Vault field (either `text` or `base64`)
+      sourceEncoding: base64
+  statuspageApiKey:
```

Comment:
how do we get this if secrets manager is not enabled?

Reply:
Right now, we don't, it is fairly required. I think I might be future-proofing too much here, I think if we used another Vault-to-k8s mechanism the values might change so that's why I was already preparing to be able to toggle between secrets-manager and some other Vault mechanism

Reply:
> I think if we used another Vault-to-k8s mechanism

I think that's highly likely down the road. So I'm totally ok with this. Just confirming there wasn't another method of getting the secret I was missing.